src/math/PercentageMath.sol

```diff
@@ -15,17 +15,17 @@ library PercentageMath {
 
     /// INTERNAL ///
 
-    /// @notice Executes a percentage addition (x * (1 + p)), rounded up.
+    /// @notice Executes a percentage addition (x * (1 + p)), rounded half up.
```

Comment:
```suggestion
    /// @notice Executes the bps-based addition (x * (1 + p)), rounded half up.
```

Reply:
I didn't apply this here because to me the operation applied here is not canonical, and not a bps-based addition
Do you have another suggestion?

Reply:
Wdym exactly ? It is divided by `PERCENTAGE_FACTOR` at the end

Reply:
I mean the canonic fixed-point addition would look like `a + b`, `a` & `b` being expected in the same base as the result

Here we are defining a new operation which is commonly defined as a percentage addition, where we add a given percentage of the input, to the input, to get the result
I'm not a fan of defining this operation as **the bps-based addition** because it can be confusing

What do you think?

Reply:
Ok I see, it can be misleading your are right. Then we can go for:
```suggestion
    /// @notice Executes a percentage bps-based addition (x * (1 + p)), rounded half up.
```